CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 0.19.1 (2022-xx-xx)
```

Comment:
I'll do a 0.20.0 friday. You can remove this entry

Reply:
Thanks, done.